app/screens/play_screen.ts

```diff
@@ -33,8 +37,8 @@ export default class PlayScreen extends Screen {
             uuid: this.uuid,
         });
 
-        play.onclick = this.click_play.bind(this);
-        pause.onclick = this.click_pause.bind(this);
```

Comment:
This `play` and `pause` variables aren't initialized, and TS complained about that.
So that's the reason why I moved them to class properties instead